.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,32 @@
+name: Feature request
+description: Suggest an idea for this project.
+title: "bixaluswds - feature request: [YOUR_TITLE]"
+labels: ["Feature request"]
+
+body:
+  - type: textarea
+    id: request
+    attributes:
+      label: Feature request
+      description: "A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]"
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the solution you'd like
+      description: "A clear and concise description of what you want to happen."
+    validations:
+      required: true
+  - type: textarea
+    id: solution
```

Comment:
it looks like this needs to be a different ID... maybe change to alternative?  here was the screen shot and link that i saw for the issue. 

[Link to issue doc](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms#body-must-have-unique-ids)


<img width="1049" alt="Screenshot 2024-09-09 at 9 16 57 AM" src="https://github.com/user-attachments/assets/ef5d6898-0a6b-434b-8444-801e6326bf8d">